source/docs/articles/wordpress/site-networks/getting-started.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Up and Running with WordPress Site Networks
+description: Get started with WordPress Site Networks. Learn about the Pantheon WordPress Site Network support, start developing, or import existing networks.
+---
+
+Once a Pantheon employee has given you access to your WordPress Site Network site, you can import an existing network or start from scratch and begin developing your project on the platform.
+
+## Migrating Site Networks to Pantheon
+
+If your network already exists, follow the instructions in the [Migrate to Pantheon: WordPress Site Networks](/docs/articles/sites/migrate/wordpress-site-networks/) doc.
+
+## Create a Network
+
+Before you begin, switch the Development environment to SFTP mode. Creating the network requires making modifications to the `wp-config.php` file using either WP-CLI or the WordPress Dashboard, which is not possible if the site's connection mode is set to Git.
```

Comment:
Have you tested modifying wp-config.php with Git? I am able to do so. Delete the second sentence?